features/credit_card_url.feature

```diff
@@ -30,3 +30,12 @@ Feature: Credit Card Form URL
 
     When I visit my profile page
     Then the new value should be stored as my Credit Card Form URL
+
+  Scenario: Vendor cannot save credit card URL with invalid format
```

Comment:
yeah let's talk about the strategy! I know the team is using cucumber and I am OK with doing that but perhaps we should define when to use cucumber and when to use unit tests / controller tests only.